### ISSUE TITLE
[Bug] Identity Logo SVG lacks focus indicator when state `:focus` added

### ISSUE THREAD

**Issue by possumbilities** [creativecommons/index-prototype#24]
## Description
Likely due to the image-mask applied to the identity logo via svg, the element lacks an appropriate focus indicator when the user navigates to it via the keyboard.

## Reproduction

1. Tab through the focusable items, via keyboard
2. See that the logo does not get an appropriate default indicator similar to the other times (i.e. menu links)

## Expectation
The logo should have the browser default focus indicator around it, or similar. Generally this would be a border/outline.

We want to keep the svg as a background image, while also correcting the `:focus` behavior, so that we can still control the color of the svg dynamically. This will likely entail adding/altering the html, likely the CSS, but we want to avoid the use of an embedded `<img>` here. 

## Resolution

- [x] I would be interested in resolving this bug.

**Comment by isabelxklee** [comment]
Hi @possumbilities! I've come up with a solution for this issue, but it looks like I don't have permission to push my branch to the repo. 👀 I've read the contribution guide that recommends contributors to push early and often, so I'm wondering if this particular repo is restricted!
